src/main.js

```diff
@@ -297,12 +297,14 @@ const CodingManagerApplication = new Lang.Class({
         let active_window_xid = active_window ? active_window.get_xid() : 0;
         let current_window_xid = current_window ? current_window.get_xid() : 0;
 
-        // try to match transient windows
-        let transient_window = active_window.get_transient();
+        if (active_window != null) {
```

Comment:
But is this restructure correct? Having no active window should result in hiding the manager window. Currently it does nothing. I'll test it.